### DISCUSSION TITLE
PimCore 6.9 Schedule problem

### DISCUSSION THREAD

**Discussion by betterapp** [pimcore/pimcore#13459] [Q&A / Support] [answered]
Hi

We use PimCore 6.9.6
When we setup shedule for publish new version object is published but active column is not set to 0 after schedule run.
Instead of this the new schedule record (with new id) is created but active column have always 1 value.

Looks like Task Dao is ok and problem is on lover level.

I will continue researching but maybe someone have solution for this ?

**Comment by blankse** [comment, marked as answer]
@betterapp This bug is fixed in Pimcore 10 (see https://github.com/pimcore/pimcore/pull/10320) Pimcore 6 has reached EOL, so this will not fixed there. I think you need to update.

**Comment by betterapp** [comment]
We still use 6.9 on some instances and can not move easily to PimCore X so we will try to use patch from one PR
